go/arrow/array/concat_test.go

```diff
@@ -78,6 +79,8 @@ func TestConcatenate(t *testing.T) {
 		{arrow.BinaryTypes.LargeString},
 		{arrow.ListOf(arrow.PrimitiveTypes.Int8)},
 		{arrow.LargeListOf(arrow.PrimitiveTypes.Int8)},
+		{arrow.ListViewOf(arrow.PrimitiveTypes.Int8)},
+		// {arrow.LargeListViewOf(arrow.PrimitiveTypes.Int8)},
```

Comment:
Should be uncommented?

Reply:
I can't gen large list-view yet and the 32-bit list view generator code is generating bogus data

Reply:
Now that I squashed all the bugs for list-view concatenation I will add the 64-bit variations.